R/getQuote.R

```diff
@@ -372,7 +372,7 @@ getQuote.av <- function(Symbols, api.key, ...) {
   }
 
   # Normalize column names and output
-  r <- r[, c("ticker", "lastSaleTimestamp", "open", "high", "low", "last", "volume")]
+  r <- r[, c("ticker", "lastsaleTimeStamp", "open", "high", "low", "last", "volume")]
```

Comment:
I'm still getting `"lastSaleTimestamp"` returned from the service. And I checked the [API docs](https://api.tiingo.com/documentation/iex), and that's what it has listed too.

@tiingo, can you comment on why @ethanbsmith is getting `"lastsaleTimeStamp"` instead of the documented behavior? The request is: `https://api.tiingo.com/iex/?tickers=SPY&token=[]`.

Reply:
![Capture](https://user-images.githubusercontent.com/24379655/69496217-f82b0280-0e8c-11ea-84fe-676418575a27.PNG)

very strange. I tried directly hitting the URL in a browser and same result. screen shot attached.

Reply:
I spoke w/ @tiingo. this is cleared up now and we can roll back 5ff4cd6. Josh, can i leave this to you so i don't have to show off my lack of git skills